client/app/src/accounts/account.controller.js

```diff
@@ -219,7 +219,7 @@
     if (!self.network.themeDark) self.network.themeDark = false
 
     // will be used in view
-    self.currentTheme = self.network.theme
+    self.currentTheme = 'default';//self.network.theme
```

Comment:
Should probably remove the commented out code here

Reply:
Thanks @spresnal . But setting theme's name before registering it caused a lot of extra watches. watching themes and colors in angularJS Material is so expensive and inefficient 
![screenshot from 2017-11-16 23-08-42](https://user-images.githubusercontent.com/5755214/32912203-8d0e995c-cb23-11e7-94bb-6422d9fc3c8a.png)
BTW, Thanks for your hint, I made some other changes, if they didn't fix it, I'll revert them. (just preserving the main fix commit)

THanks :)